server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/GetMailboxesMethodTest.java

```diff
@@ -681,6 +681,28 @@ public void getMailboxesShouldReturnSharedMailboxesWithRead() throws Exception {
             .body(ARGUMENTS + ".list.id", hasItems(mailboxId.serialize()));
     }
 
+    @Test
+    public void getMailboxesShouldReturnSharedMailboxesWithoutRole() throws Exception {
```

Comment:
Can we extract the Draft bugfix as a separate PR (please?)

Reply:
=> #3406